TEStribute/distance.py

```diff
@@ -0,0 +1,27 @@
+import geopy.distance
+from ip2geotools.databases.noncommercial import DbIpCity
+import socket
+
+
+def ReturnDistance(ip1, ip2):
+
+    ip1 = DbIpCity.get(socket.gethostbyname(ip1), api_key="free")
```

Comment:
The API key is something that could be parametrized and configured in some setup file for the `TEStribute` package. This is **not** what should go in the test configs file though, as it is not something we would expect the calling code to set up. It's the choice of whoever installs/uses `TEStribute` in their service.

This doesn't have high priority, but perhaps it would be good to add an issue.